modules/Sources/Dependencies/AddressBookClient/AddressBookLiveKey.swift

```diff
@@ -54,13 +61,14 @@ extension AddressBookClient: DependencyKey {
                     let encryptedFileURL = documentsDirectory.appendingPathComponent(try AddressBookClient.filenameForEncryptedFile())
```

Comment:
Nit: on line 54, "locale storage" should be "local storage".

Reply:
Updated